components/core/Main.py

```diff
@@ -11,7 +11,18 @@ def main():
 
 
 if __name__ == '__main__':
-    logging.basicConfig(filename="./debug-server.log", level=logging.DEBUG)
-    logging.warning("INFO: Bassa starting")
+    logging.basicConfig(
+        filename="./debug-server.log",
+        level=logging.DEBUG,
+        format='%(asctime)s %(levelname)-8s %(message)s',
+        datefmt='%m-%d %H:%M'
+    )
+
+    console = logging.StreamHandler()
+    formatter = logging.Formatter('%(levelname)-8s %(message)s')
+    console.setFormatter(formatter)
+    logging.getLogger('').addHandler(console)
```

Comment:
Did you check that after adding a console to stream, what extra is getting logged?
I am super excited to see what magic this code snippet did.

Reply:
https://imgur.com/a/2mjpmu9
An example when you have wrong database credentials :)
This will be much useful in other scenarios aswell.
If without this stream you will see nothing in the console.

Reply:
ok great thanks @xlight05 :)